libraries/core/src/main/kotlin/com/bumble/appyx/core/node/Node.kt

```diff
@@ -74,9 +76,12 @@ abstract class Node(
         }
 
     private val lifecycleRegistry = LifecycleRegistry(this)
-
     private var wasBuilt = false
 
+    override val requestCodeClientId: String =
+        buildContext.savedStateMap?.get(RequestCodeClientIdKey) as String?
```

Comment:
Let's introduce public `id` property for node.

Reply:
https://github.com/bumble-tech/appyx/pull/175/files#diff-a57bab82c13427fb82ff9ac8efdc79f337a988817310bb19a006a9a276df04ddR54